src/data/actions/assets.js

```diff
@@ -1,7 +1,10 @@
-import { getAssets } from '../../api/client';
+import * as clientApi from '../../api/client';
```

Comment:
I thought this was a little more clear to know where the getAssets function is coming from by referring to the clientApi explicitly, but I don't feel strongly about it. It was also to allow for me to use the deleteAsset function name in this file and in the client api without name conflicts.